src/sentry/conf/server.py

```diff
@@ -2331,14 +2331,6 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 
 SENTRY_METRICS_INDEXER_ENABLE_SLICED_PRODUCER = False
 
-# Release Health
-SENTRY_RELEASE_HEALTH = "sentry.release_health.metrics.MetricsReleaseHealthBackend"
-SENTRY_RELEASE_HEALTH_OPTIONS: dict[str, Any] = {}
-
-# Release Monitor
-SENTRY_RELEASE_MONITOR = "sentry.release_health.release_monitor.metrics.MetricReleaseMonitorBackend"
-SENTRY_RELEASE_MONITOR_OPTIONS: dict[str, Any] = {}
```

Comment:
Should we keep these around? We typically always have a setting for anything using a `LazyServiceWrapper`

Reply:
Don't think so - otherwise we are potentially allow a user to override something when doing so will result in broken functionality. I actually wanted to remove the LazyServiceWrapper entirely in this PR too but it was a bit too involved.

Reply:
I think we should keep the `LazyServiceWrapper` in place, we generally use this so that we can replace backends in the future if we need to. Don't mind too much about the settings though